### DISCUSSION TITLE
Development Server Not Loading

### DISCUSSION THREAD

**Discussion by seancrossettie** [nss-evening-cohort-14/evening-client-side#244] [Help Me!] [answered]
Name
Sean Rossettie

Have you searched for other help tickets? 
Yes

Describe your issue
My development server is not loading and my computer sounds like it's going to explode every time I 'npm start'. 

What have you tried?
I have tried restarting my machine a few times to no avail. I am getting a warning from my terminal saying the following:

``
found 81 vulnerabilities (80 moderate, 1 high) in 2110 scanned packages
  81 vulnerabilities require manual review. See the full report for details.
``
here is the _vulnerability:_
<img width="589" alt="Screen Shot 2021-05-24 at 7 02 29 PM" src="https://user-images.githubusercontent.com/76828201/119420644-98502a80-bcc2-11eb-95e9-48523dc6fa3c.png">

I am currently looking up what this could mean on Google but wanted to see if the instructors team could help me out in case you all know whats going on. 
Thank you!

**Comment by drteresavasquez** [comment, marked as answer]
@seancrossettie can you add the solution?

**Comment by seancrossettie** [comment]
Turns out the solution to this was to checkout to a previous branch and compare changes I made across branches on Github. I just pulled down an earlier branch and worked from there!
